doc/src/modules/stats.rst

```diff
@@ -97,9 +100,22 @@ Joint Types
 
 Stochastic Processes
 --------------------
-.. autofunction:: DiscreteMarkovChain
-.. autofunction:: ContinuousMarkovChain
-.. autofunction:: BernoulliProcess
+.. autoclass:: DiscreteMarkovChain
+.. autoclass:: ContinuousMarkovChain
+.. autoclass:: BernoulliProcess
+.. autoclass:: PoissonProcess
+.. autoclass:: WienerProcess
+.. autoclass:: GammaProcess
```

Comment:
The reference, https://www.probabilitycourse.com in these three processes should point to their respective pages under the website, for now it is just landing to the home page.

Reply:
Okay!